ts/Grid/Core/Table/Header/HeaderCell.ts

```diff
@@ -156,43 +157,48 @@ class HeaderCell extends Cell {
 
     /**
      * Render the cell container.
+     *
+     * @param filtering
+     * Whether the cell is part of the filtering row.
      */
-    public override render(): void {
+    public override render(filtering?: boolean): void {
```

Comment:
IMO isFiltered would be more meaningful. The `filtering` sounds like an option with props.

Just for consideration...